HellBoy-OP/Plugins
# Railway Support (#26)

* Create Dockerfile

* Update start.sh

* Testing regex for command clashes (#1)

* Update _core.py

* Update _helper.py

* Update insta.py

* Update _helper.py

* Update _helper.py

* Update _helper.py

* Update _helper.py

* Update _inhelp.py

* Regex fix (A) (#2)

* Regex Fix (B, C) (#3)

* Regex Fix (D) (#4)

* Update anime.py

* Update Procfile

* Update README.md

* Lets see (#8)

* Regex fix and updates (E, F) (#5)

* Update forcesub.py

* Update forcesub.py

* Update forcesub.py

* Update (#6)

* Update msgs.py

* Updates (#7)

* Create Aptfile

* Update echo.py

* Update start.sh

* Update start.sh

* Update forcesub.py

* Update forcesub.py

* Delete Dockerfile

* Update requirements.txt

* Update songs.py

* Update youtube.py

* Update start.sh
